frontend/src/js/route-config/route-config.js

```diff
@@ -127,6 +127,16 @@
             authenticate: true
         };
 
+        var invitation = {
+            name: "web.invitation",
+            parent: "web",
+            url: "/invitation/:hash/:encoding",
```

Comment:
HI @isht3 , please change the name of hash and encoding to something meaningful and non redundant. Actually hash and encoding are very common words.

Reply:
@aka-jain How about `email_hash` and, `user_enc`?